background.ts

```diff
@@ -1,6 +1,7 @@
 import { Console } from 'console';
 import fs from 'fs';
 import path from 'path';
+import process from 'process';
```

Comment:
`process` should be a built-in global already?

Reply:
Didn't know that. Should that line also be removed from `scripts/hyperkit.mjs`

Reply:
Yeah, there's no point in importing `process`.